### PR TITLE
add skprocrustes

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://github.com/melissawm/skprocrustes

### CODE REVIEW

File: recipes/scikit-procrustes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - matplotlib
+
+test:
+  imports:
+    - skprocrustes
+
+about:
+  home: https://github.com/melissawm/skprocrustes/
+  license: BSD 2-Clause
+  # license_file: LICENSE.rst
```
Comment (ocefpaf) [inline recipes/scikit-procrustes/meta.yaml]: @melissawm the license file is not in the source distribution.
I can send a PR to you later to fix this.

File: recipes/scikit-procrustes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  summary: 'This is a scikit for the Weighted Orthogonal Procrustes Problem.'
+
+extra:
+  recipe-maintainers:
+    - melissawm
```
Comment (ocefpaf) [inline recipes/scikit-procrustes/meta.yaml]: Do you want to be listed as a maintainer of the `conda` recipe?
Here is what it entails: http://conda-forge.github.io/docs/recipe.html#maintainer-role

File: recipes/scikit-procrustes/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.12
+    - scipy
```
Comment (synapticarbors) [inline recipes/scikit-procrustes/meta.yaml]: Are `scipy` and `matplotlib` required to build the package? If not, please remove them.

Comment (ocefpaf) [inline recipes/scikit-procrustes/meta.yaml]: At the moment, yes. It will be fixed upstream.

**Pull request merged**